simulator/src/main.rs

```diff
@@ -36,21 +40,54 @@ fn main() -> Result<()> {
     // Force interface to run on the last possible core
     core_affinity::set_for_current(cores.pop_back().unwrap());
 
+    let node_core = cores.pop_front().unwrap();
```

Comment:
I would not spawn the server on the simulator at all, tbh. Just assume the user has started it elsewhere.